lib/morpheus.gi

```diff
@@ -3001,3 +3001,42 @@ local cl,cnt,bg,bw,bo,bi,k,gens,go,imgs,params,emb,clg,sg,vsu,c,i;
   Info(InfoMorph,1,Length(emb)," found -> ",Length(cl)," homs");
   return cl;
 end);
+
+# TODO: Maybe use Babai-Szemeredi Cube-method, see Permutation Group 
+# Algorithms by Seress, Section 4.4.
+BindGlobal("RepresentativesOuterAutomorphismGroup", 
+function(T, args...)
+  local order_out, out_reps, candidate, rep, i;
+  if Length(aut) = 1 then 
```

Comment:
`aut` -> `args`

Reply:
Updated and changed the subsequent calls to `Length(aut)` to `Length(args)` as well.